src/c++/uds/src/uds/errors.h

```diff
@@ -21,7 +21,8 @@
 /* Valid status codes for internal UDS functions. */
 enum uds_status_codes {
 	/* Successful return */
-	UDS_SUCCESS = 0,
+	VDO_SUCCESS = 0,
```

Comment:
This feels weird to me. In my mind, I was assuming there was a common include that defined VDO_SUCCESS = 0 and then we would include that everywhere it was needed, including the uds ones. I know that sort of goes against how we have uds feed into vdo currently, but is that something we are looking at changing down the road?

Reply:
I wasn't suggesting a single file. I assumed we might have a very common file somewhere. If thats errors.h, then I might consider moving VDO_SUCCESS = 0 out of the uds_status_codes enum and put it at the top of the file.

Reply:
Hm, could do.

Reply:
very